elytron/src/main/java/org/wildfly/extension/elytron/CredentialStoreService.java

```diff
@@ -114,6 +114,9 @@ public void start(StartContext startContext) throws StartException {
             credentialStoreAttributes.put(CS_LOCATION_ATTRIBUTE, loc.toAbsolutePath().toString());
             credentialStore = getCredentialStoreInstance();
             credentialStore.initialize(credentialStoreAttributes, resolveCredentialStoreProtectionParameter(), otherProviders.getOptionalValue());
+            if ( credentialStoreAttributes.get(ElytronDescriptionConstants.CREATE).equals("true") && !loc.toFile().exists() ){
+                credentialStore.flush();
```

Comment:
@pskopek Never mind. Looking at #2308 I see that it's about keeping the FS store and the in-memory one in sync. And that's not an issue here as this call is not storing anything to the in-memory store.